### DISCUSSION TITLE
Links for email addresses don't work

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#22] [Report a bug] [answered]
With xxx@yyy.com the Emai address is displayed as a link. 

With the following email address does not work: 
xxx.xxx@yyy.com 
The point between xxx.yyy is not accepted and is displayed incorrectly. 

Thank you 
Peter

**Comment by markseuffert** [comment, marked as answer]
You are right! The automatic linking was too strict, there's an improved version in latest update. 

Thanks Peter for testing and reporting bugs! :smile:

**Comment by ghost** [comment]
Works! 

Thanks a lot 
Peter
